src/mmw/apps/geoprocessing_api/calcs.py

```diff
@@ -187,25 +187,30 @@ def calculate_avg_slope(slope, length):
     }
 
 
-def point_source_pollution(geojson):
+def point_source_pollution(geojson, datasource=None):
     """
     Given a GeoJSON shape, retrieve point source pollution data
-    from the `ms_pointsource` or `ms_pointsource_drb` table to display
-    in the Analyze tab.
+    from the `ms_pointsource`, `ms_pointsource_drb` or
+    `ms_pointsource_pa` table to display in the Analyze tab.
+    Use `ms_pointsource_drb` table if AoI within DRB, otherwise
+    retrieves from `ms_pointsource`. If valid datasource requested,
+    override source with respective table.
 
     Returns a dictionary to append to the outgoing JSON for analysis
     results.
     """
     geom = GEOSGeometry(geojson, srid=4326)
-    drb = geom.within(DRB)
-    table_name = get_point_source_table(drb)
-    sql = f'''
+    table_name = get_point_source_table(geom.within(DRB), datasource)
+    # facilityname col exists for PA and DRB tables
+    has_facilityname = table_name.endswith("_pa") or table_name.endswith(
+        "_drb"
+    )
+    sql = f"""
```

Comment:
The project convention is to use single quotes unless otherwise necessary

```suggestion
    has_facilityname = table_name.endswith('_pa') or table_name.endswith(
        '_drb'
    )
    sql = f'''
```